src/atopile/cli/common.py

```diff
@@ -140,21 +136,14 @@ def create_build_contexts(
             raise errors.UserBadParameterError(
                 f"The file you have specified does not exist: {entry_arg_file_path}"
             )
-        if not entry_arg_file_path.is_file():
-            raise errors.UserBadParameterError(
-                "The path you're building with the --standalone"
-                f" option must be a file {entry_arg_file_path}"
-            )
-        if not address.get_entry_section(entry):
-            raise errors.UserBadParameterError(
-                "You must specify what to build within a file to build with the"
-                " --standalone option"
-            )
 
         project_config = atopile.config.ProjectConfig(
             location=Path.cwd(),
             ato_version=f"^{version.get_installed_atopile_version()}",
-            paths=atopile.config.ProjectPaths(layout=Path.cwd() / "standalone"),
+            paths=atopile.config.ProjectPaths(
```

Comment:
This config/context concept has legs, but we need to reconcile creating them cleanly. There are currently bits you want duplicated or missing in one place or the other